frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/backing-storage-step/backing-storage-step.tsx

```diff
@@ -78,7 +77,6 @@ const StorageClassSelection: React.FC<StorageClassSelectionProps> = ({ dispatch,
         noSelection
         onChange={onStorageClassSelect}
         selectedKey={selected.name}
-        filter={filterSCWithoutNoProv}
```

Comment:
Can we now make internal mode cluster with `no-prov` storage-class as well ??

Reply:
yeah , we have removed notion of internal and external mode.